core/templates/services/guppy-initialization.service.ts

```diff
@@ -35,16 +37,30 @@ export class GuppyInitializationService {
   private guppyInstances: Array<GuppyObject> = [];
   private onScreenKeyboardShown = false;
 
-  init(guppyDivClassName: string): void {
+  init(guppyDivClassName: string, initialValue = ''): void {
     this.onScreenKeyboardShown = false;
     let guppyDivs = document.querySelectorAll('.' + guppyDivClassName);
     let divId, guppyInstance;
+    let mathInteractionsService = new MathInteractionsService();
     for (let i = 0; i < guppyDivs.length; i++) {
       divId = 'guppy_' + Math.floor(Math.random() * 100000000);
       // Dynamically assigns a unique id to the guppy div.
       guppyDivs[i].setAttribute('id', divId);
       // Create a new guppy instance for that div.
       guppyInstance = new Guppy(divId, {});
+
+      // Initialize if with a value for the creator's view.
+      if (guppyDivClassName === 'guppy-div-creator') {
+        initialValue = initialValue.replace(/abs\(/g, 'absolutevalue(');
```

Comment:
Are these the only two cases? It should actually be done more declaratively, with a constant mapping and a for loop (and that constant should be used everywhere it needs to be).

Reply:
Yes, I've tested all the functions that we support and these are the only two cases that need to be handled.
Now that this code is only present in this file, I don't need to add it in a constant, right?